k8s/k8s.go

```diff
@@ -637,7 +637,7 @@ func getK8sPodInfo(client *kubernetes.Clientset, podName, podNamespace string) (
 		labels = make(map[string]string)
 	}
 
-	labels["calico/k8s_ns"] = podNamespace
+	labels[apiv2.LabelNamespace] = podNamespace
```

Comment:
This will get automatically added by libcalico, right?

I don't think we need this line here any more, though I'm happy to leave it since it won't hurt anything.

Reply:
oh yes it will